src/chains/ethereum/ethereum/src/api.ts

```diff
@@ -2080,16 +2080,15 @@ export default class EthereumApi implements Api {
           (blockLogs: BlockLogs) => {
             // TODO: move the JSON stringification closer to where the message
             // is actually sent to the listener
```

Comment:
looks like we don't need the comment anymore?

Reply:
Yep, good find!